Makefile.Common

```diff
@@ -9,6 +9,7 @@ ALL_SRC_AND_DOC := $(shell find . \( -name "*.md" -o -name "*.go" -o -name "*.ya
 ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))))
 # ALL_TEST_DIRS includes ./* dirs (excludes . dir)
 ALL_TEST_DIRS := $(shell find . -type f -name "go.mod" -exec dirname {} \; | sort | egrep  '^./' )
+ALL_TEST_DIRS := . $(ALL_TEST_DIRS)
```

Comment:
remove egrep on line 11 should include . dir. I had excluded . dir as I was seeing some issue with go 1.13 but I don't see that now. 
Please update the comment on line 10 accordingly.

Reply:
Thanks. Updated.